content/en/docs/setup/version-skew-policy.md

```diff
@@ -0,0 +1,127 @@
+or on an as-needed basis as determined by the [patch release manager](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/patch-release-manager/README.md#release-timing).
+
+With minor releases happening approximately every three months,
+that means each minor release is maintained for approximately nine months.
+
+## Supported version skew
+
+### `kube-apiserver`
+
+In multi-server clusters, the newest and oldest `kube-apiserver` instances must be within one minor version.
+
+Example:
+
+* newest `kube-apiserver` is at `1.13`
+* other `kube-apiserver` instances are supported at `1.13` and `1.12`
+
+### `kubelet`
+
+`kubelet` must not be newer than `kube-apiserver`, and may be up to two minor versions older.
+
+Example:
+
+* `kube-apiserver` is at `1.13`
+* `kubelet` is supported at `1.13`, `1.12`, and `1.11`
+
+Note that if version skew exists between `kube-apiserver` instances in a multi-server cluster, this narrows the allowed `kubelet` versions.
+
+Example:
+
+* `kube-apiserver` instances are at `1.13` and `1.12`
+* `kubelet` is supported at `1.12`, and `1.11` (`1.13` is not supported because that would be newer than the `kube-apiserver` instance at version `1.12`)
+
+### `kube-controller-manager` and `kube-scheduler`
+
+`kube-controller-manager` and `kube-scheduler` must not be newer than the `kube-apiserver` instances they communicate with. They are expected to match the `kube-apiserver` minor version, but may be up to one minor version older (to allow live upgrades).
+
+Example:
+
+* `kube-apiserver` is at `1.13`
+* `kube-controller-manager` and `kube-scheduler` are supported at `1.13` and `1.12`
+
+Note that if version skew exists between `kube-apiserver` instances in a multi-server cluster, and `kube-controller-manager` or `kube-scheduler` can communicate with any `kube-apiserver` instance in the cluster (for example, via a load balancer), this narrows the allowed `kube-controller-manager` or `kube-scheduler` versions.
+
+Example:
+
+* `kube-apiserver` instances are at `1.13` and `1.12`
+* `kube-controller-manager` and `kube-scheduler` communicate with a load balancer that can route to any `kube-apiserver` instance
+* `kube-controller-manager` and `kube-scheduler` are supported at `1.12` (`1.13` is not supported because that would be newer than the `kube-apiserver` instance at version `1.12`)
+
+### `kubectl`
+
+`kubectl` is supported within one minor version (older or newer) of `kube-apiserver`.
+
+Example:
+
+* `kube-apiserver` is at `1.13`
+* `kubectl` is supported at `1.14`, `1.13`, and `1.12`
+
+Note that if version skew exists between `kube-apiserver` instances in a multi-server cluster, this narrows the supported `kubectl` versions.
+
+Example:
+
+* `kube-apiserver` instances are at `1.13` and `1.12`
+* `kubectl` is supported at `1.13` and `1.12` (other versions would be more than one minor version skewed from one of the `kube-apiserver` components)
+
+## Supported component upgrade order
```

Comment:
This is awesome and very well written. However, I'm not sure we actually have testing to verify that this specific ordering works.

Reply:
this was intended to describe the order followed by upgrade scripts (including the ones used in CI upgrade tests). if you see inaccuracies, definitely point them out